### DISCUSSION TITLE
How to set unicode characters for pseudo classes in config?

### DISCUSSION THREAD

**Discussion by lessless** [tailwindlabs/tailwindcss#8607] [Help] [answered]
Hi, 

We are trying to set a Unicode character for a pseudo-classes in the tailwind config: 

```
module.exports = {
  theme: {
    extend: {
      content: {
        "chevron-rb": '"\uE970"',
        
```

I'm not sure how wrong is this, the "solution" came up after a certain amount of trials and errors.  Still, it does not fully address the problem because this symbol comes from the icon font.

Of course, it can be solved by specifying font for pseudo-classes, but maybe there is a better solution?

**Comment by rohmann** [comment, marked as answer]
Hi @lessless,

If you're looking to setup a custom icon library where you can do something like `.custom-icon-chevron-r` you could use the `matchComponents` API. This technique also provides a place to include your font family and any other CSS needed to setup the icons.

Here's an example with most unicode values, and just copy/pasting characters. Personally I prefer the latter so I can see what it looks like in the code, but obviously that only works when the character is included in the font of your IDE.

https://play.tailwindcss.com/O1aCgwU4Fr?file=config

```js
const plugin = require('tailwindcss/plugin')

module.exports = {
  theme: {
    iconFont: {
      'chevron-l': '\u2039',
      'chevron-r': '\u203A',
      'chevron-r2': '›',
      'chevron-l2': '‹'
    },
  },
  
  plugins: [
    plugin(function ({ matchComponents, theme, e }) {
      matchComponents(
        {
          'custom-icon': (value) => ({
            'font-family': 'Icon Font',
            '&:after': {
              content: `"${value}"`,
            },
          }),
        },
        { values: theme('iconFont') }
      )
    }),
  ],
}
```

**Comment by ebartan** [comment]
handy reference: https://www.htmlsymbols.xyz/unicode/U+00F7
